py4cast/lightning.py

```diff
@@ -918,14 +993,30 @@ def test_step(self, batch: ItemBatch, batch_idx):
                 prog_bar=False,
             )
 
+        self.test_step_logging(batch, prediction, target, mask)
+
+    def test_step_logging(
+        self,
+        batch: ItemBatch,
+        prediction: NamedTensor,
+        target: NamedTensor,
+        mask: torch.Tensor,
+    ):
+        """
+        Save metrics and plots of test to the tensorboard
+        """
```

Comment:
Docstring formating.
```suggestion
        """Saves metrics and plots of test to the tensorboard
        """
```